frontend/interfaces/label.ts

```diff
@@ -3,7 +3,7 @@ import PropTypes from "prop-types";
 export default PropTypes.shape({
   created_at: PropTypes.string,
   updated_at: PropTypes.string,
-  id: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+  id: PropTypes.oneOfType([PropTypes.number]),
```

Comment:
I think this could just be `PropTypes.number` but I'm not concerned since this object isn't necessary with the interface.